Makefile.e2e

```diff
@@ -1,7 +1,10 @@
 .PHONY: skaffold.validate
 skaffold.validate: kubectl_context := $(shell kubectl config current-context)
 skaffold.validate:
-	@if [[ ! "minikube,docker-for-desktop,docker-desktop" =~ .*"$(kubectl_context)".* ]]; then \
+
+	@if [[ "$(kubectl_context)" =~ .*"kind".* ]]; then \
+		true; \
+	elif [[ ! "minikube,docker-for-desktop,docker-desktop" =~ .*"$(kubectl_context)".* ]]; then \
```

Comment:
Wouldn't it work if we just added `kind` to this list of words?

Reply:
Nah, kind gives you a context name based on the name of your cluster.

default cluster name would be ‘kind’ and context would be ‘kind-kind’. But if someone names it diffeeenrly than the test breaks.

I moved it to a regular expression check for this reason.  I think it’s always kind-* if you are using a kind cluster. 

My bash isn’t that great either so I’m open for a better way of structuring this code.

Reply:
Could you, please, describe your rationale as a comment in the code? That would be helpful.